analyzer.go

```diff
@@ -56,6 +56,9 @@ func (a *Analyzer) analyzeLayers(appMeta LayersMetadata, cache Cache) error {
 	var cacheMeta CacheMetadata
 	if cache != nil {
 		var err error
+		if !cache.Exists() {
+			a.Logger.Info("cache does not exist")
```

Comment:
What is the motivation behind adding this? I worry slightly that this may look like a _bad thing_ if it is expected on a first build. Also we should capitalize log statement e.g. `Cache does not exist`.

If we really want this I am inclined to say we should check whether the metadata is empty (or maybe return a bool in addition to the metadata?) and then then the users the cache is empty or unpopulated. "Exists" feels a bit confusing to me because, in the case of a cache volume for example the directory/volume may exist, even if it has no stored data.

Reply:
This came up in https://github.com/buildpacks/lifecycle/pull/409#discussion_r481233694 and just rode along with this PR.